dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -126,3 +126,49 @@ CALL drop_column_t_ds_alert_plugin_instance;
 DROP PROCEDURE drop_column_t_ds_alert_plugin_instance;
 
 DROP TABLE IF EXISTS `t_ds_trigger_relation`;
+
+-- Rename tables and fields from process to workflow
+ALTER TABLE t_ds_alert change process_definition_code workflow_definition_code bigint(20);
+ALTER TABLE t_ds_alert change process_instance_id workflow_instance_id int(11);
+
+ALTER TABLE t_ds_command change process_definition_code workflow_definition_code bigint(20);
+ALTER TABLE t_ds_command change process_instance_priority workflow_instance_priority int(11);
+ALTER TABLE t_ds_command change process_instance_id workflow_instance_id int(11);
+ALTER TABLE t_ds_command change process_definition_version workflow_definition_version int(11);
+
+ALTER TABLE t_ds_error_command change process_definition_code workflow_definition_code bigint(20);
+ALTER TABLE t_ds_error_command change process_instance_priority workflow_instance_priority int(11);
+ALTER TABLE t_ds_error_command change process_instance_id workflow_instance_id int(11);
+ALTER TABLE t_ds_error_command change process_definition_version workflow_definition_version int(11);
+
+ALTER TABLE t_ds_process_task_relation change process_definition_version workflow_definition_version int(11);
+ALTER TABLE t_ds_process_task_relation change process_definition_code workflow_definition_code bigint(20);
+
+ALTER TABLE t_ds_process_task_relation_log change process_definition_version workflow_definition_version int(11);
+ALTER TABLE t_ds_process_task_relation_log change process_definition_code workflow_definition_code bigint(20);
+
+ALTER TABLE t_ds_process_instance change process_definition_code workflow_definition_code bigint(20);
+ALTER TABLE t_ds_process_instance change process_definition_version workflow_definition_version int(11);
+ALTER TABLE t_ds_process_instance change is_sub_process is_sub_workflow int(11);
+ALTER TABLE t_ds_process_instance change process_instance_priority workflow_instance_priority int(11);
```

Comment:
It's better to use  procedure to make sure the upgrade is idempotent

Reply:
Done.